x/accounts/defaults/asset/depinject/depinject.go

```diff
@@ -0,0 +1,10 @@
+package assetdepinject
+
+import (
+	"cosmossdk.io/x/accounts/accountstd"
+	asset "cosmossdk.io/x/accounts/defaults/asset"
+)
+
+func ProvideAccount() accountstd.DepinjectAccount {
```

Comment:
Looks like you won't be able to init custom mint/burn/transfer function with this